kernel/bench.c

```diff
@@ -126,12 +126,13 @@ int run_suite(testcase_t *tests)
   for (i = 0; i < num; i++) {
     testresult_t result;
     run_benchmark(&tests[i], &result);
-    if(enable_printf) print_result(&tests[i], &result);
-
+    if(enable_printf && (get_core_id()==0))
+      print_result(&tests[i], &result);    
     errors += result.errors;
   }
-
-  print_summary(errors);
+  
+  if(get_core_id() == 0)
+    print_summary(errors);
 
```

Comment:
Can you provide some reasoning to why this is necessary?

Reply:
It is not necessary. Right now, all the cores print the same exact summary, as they all get the number of cycles from the same timer and `errors` is a shared variable. That's why I thought to simply have one core printing it but I can remove the `if(get_core_id()==0)`  :smile:

Reply:
@bluewww, any inputs on this? :rocket:

Reply:
I think this needs to be removed